share/availability/light/sample.go

```diff
@@ -1,104 +1,41 @@
-// TODO(@Wondertan): Instead of doing sampling over the coordinates do a random walk over NMT trees.
 package light
 
 import (
 	crand "crypto/rand"
-	"encoding/binary"
-	"errors"
 	"math/big"
+
+	"golang.org/x/exp/maps"
 )
 
-// Sample is a point in 2D space over square.
+// Sample represents a coordinate in a 2D data square.
 type Sample struct {
```

Comment:
Let's make it private while we are on it. Can't recall it being used elsewhere. Please check

Reply:
It would be used externally by `pruner` pkg very soon.

Reply:
Kk